src/helm/benchmark/run.py

```diff
@@ -122,6 +133,18 @@ def add_run_args(parser: argparse.ArgumentParser):
         default=None,
         help="Skip creation of instances (basically do nothing but just parse everything).",
     )
+    parser.add_argument(
+        "--save-instances-only",
+        action="store_true",
+        default=None,
+        help="Generate and save instances for scenario ONLY (i.e. do not evaluate models on instances).",
+    )
+    parser.add_argument(
+        "--read-instances-from-file",
+        action="store_true",
+        default=None,
+        help="Read instances from already generated instances.json file (if it exists)",
```

Comment:
We'll have two kinds of `instances.json` files after this (one under scenarios and one under runs) so this is ambiguous... We eventually want to get rid of the one under runs. For now, let's disambiguate for saying `the already generated instances.json file for the AdapterSpec`.

Also drop `if it exists` because the current behavior is to simply error out if the file does not exist, which I think should be the intended behavior.

Reply:
Good point. Changed file name to input_instances.json and dropped `if it exists` comment.